packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -18,9 +18,10 @@ type DefaultNotificationProps = {
 
 export const DefaultNotification = (props: DefaultNotificationProps) => {
   const style = useStyle();
-  const { t, locale } = useLocalization();
+  const { locale } = useLocalization();
   const { status } = useInboxContext();
   const [minutesPassed, setMinutesPassed] = createSignal(0);
+  const { t } = useLocalization();
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** you have this hook already defined above, please destructure the `t` in the one above